_posts/2024-04-25-WildFly32-Released.adoc

```diff
@@ -0,0 +1,324 @@
+
+==== Videos
+
+* https://www.youtube.com/watch?v=kt8pTDmTitw[Introduction to WildFly Glow]
+* https://www.youtube.com/watch?v=GkfSHiqyBDU[WildFly Glow, direct deployment to OpenShift]
+
+Also, keep an out here or on the link:https://www.youtube.com/@WildFlyAS[WildFly channel on YouTube channel] for an upcoming post from Jean Francois on using WildFly Glow to help with automatic connection to a database when deploying on OpenShift.
+
+=== User Guides
+
+We've added a new https://www.wildfly.org/guides[Guides] page to the wildly.org site. Each guide will show the steps to accomplish a specific, focused task, with links to guides showing any prerequisites and to guides for related tasks. This is something WildFly has long needed, and we're very excited to see it happening! We're now up to 10 guides in a variety of topic areas. Please have a look and give us your feedback and suggestions for other guides you'd like to see.
+
+=== Individual Features
+
+There a number of new individual features in WildFly 32, but before getting into the individual items I want to highlight again the capabilities introduced in WildFly 31 to introduce features at link:https://docs.wildfly.org/32/Admin_Guide.html#Feature_stability_levels[different stability levels]. Features can be introduced at one of four stability levels -- `experimental`, `preview`, `community` or `default` -- with the ideal outcome being that we promote them in subsequent releases to higher levels. The goal here is to allow users who want to look at features in earlier stages of the development lifecycle to easily do so, without leaving users who are not interested in that in a situation where they may inadvertently use those features.
+
+We introduced this capability in WildFly 31, and added  link:https://docs.wildfly.org/wildfly-proposals/core/WFCORE-4758_Simple_config_export_for_a_server_as_an_attachment_for_standalone_or_domain_mode.html[one feature] at `community` stability, but in WildFly 32 we've significantly expanded our use of the concept, and link:https://docs.wildfly.org/wildfly-proposals/wf-galleon/WFLY-19021-Stability_In_Provisioning.html[added support in our provisioning tooling] for it.
+
+I'll talk more about feature stability levels <<feature_stability,below>>, but first let's talk about the new features.
+
+==== Security
+
+* For outbound requests, we've added support for an `SSLContext` that can https://docs.wildfly.org/wildfly-proposals/elytron/WFLY-13762_ssl_context_based_on_peer_info.html[dynamically delegate to different SSL contexts based on destination's host and port]. This feature is provided at the `community` stability level.
+* We've made it possible to https://docs.wildfly.org/wildfly-proposals/elytron/ELY-2544-jaas-realm-via-custom-realm.html[use a `JaasSecurityRealm` via a custom-realm resource]. This feature is provided at the `default` stability level.
```

Comment:
We can omit this jaas-realm feature from the blog post I think as mentioned [here](https://github.com/wildfly/wildfly-proposals/pull/529/files#r1394648160) since this is being deprecated right away.

Reply:
@fjuma -- Fixed.